hi-ml-multimodal/test_multimodal/image/model/test_model.py

```diff
@@ -159,20 +121,20 @@ def test_hubconf() -> None:
         assert torch.allclose(value_hub, value_himl)
 
 
-def test_restore_training_mode() -> None:
-    model = torch.nn.Conv2d(3, 2, 3)
-    assert model.training
+def test_multi_image_model() -> None:
+    joint_feature_size = 4
+    with pytest.raises(AssertionError) as ex:
```

Comment:
Small one: You could use the `match` argument to `pytest.raises`
https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions

Reply:
Thanks, didn't know about it. Updated.